src/services/MQTTRoomService/MQTTRoomService.ts

```
@@ -0,0 +1,115 @@
+
+    return new Promise<T>((resolve, reject) => {
+      this.messagePromises[messageId] = { resolve, reject };
+
+      worker.postMessage({ functionName, args, messageId });
+
+      const handleAbort = () => {
+        // Note: As of now the worker & Rust service do not yet support
+        // this action, but it should be safely ignored
+        worker.postMessage({ messageId, action: "abort" });
+        reject(new Error("Aborted"));
+        delete this.messagePromises[messageId];
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)